usr/share/rear/conf/default.conf

```diff
@@ -2169,6 +2169,25 @@ COPY_AS_IS_NBKDC=()
 COPY_AS_IS_EXCLUDE_NBKDC=()
 ####
 
+####
+# BACKUP=NFS_SERVER
+##
+# Note: This is for restore via NFS server, where another computer can connect to the rescue system via NFS and push
+#       the backup files onto the disks after ReaR prepared and formatted them.
+#
+#       This method has no backup functionality, only restore. And it requires interaction with an external system to push the files.
+#
+#       ReaR will wait for a signal file to appear to indicate that the file transfer is complete.
+#
+REQUIRED_PROGS_NFS_SERVER=("rpc.nfsd" "rpc.mountd" "exportfs" "ss")
+NFS_SERVER_EXPORT_OPTS="rw,no_root_squash,async,no_subtree_check"
+NFS_SERVER_TRUSTED="*"
```

Comment:
```suggestion
# Trusted system that can mount the recovery system, takes a single argument as described in exports(5)
NFS_SERVER_TRUSTED="127.0.0.1"
```

Reply:
One other thing @codefritzel, can we please turn this into an array and support setting multiple trusted clients?

Like for example
```
NFS_SERVER_TRUSTED=( client1 192.168.10/24 )
```
would allow both a single client and an IP range.

This is even backwards compatible for users who set it to a single value :smile:

Reply:
No, it should start from the same `NFS_SERVER` to show up in `dump`. We can go for `NFS_SERVER_TRUSTED_CLIENTS` or `NFS_SERVER_ALLOWED_CLIENTS` which would also denote that this is about the clients that may connect.

Reply:
Yes,
"NFS_SERVER_TRUSTED" means a single NFS server that is trused.
In contrast "NFS_SERVER_TRUSTED_CLIENTS"
or NFS_SERVER_ALLOWED_CLIENTS" mean one or more clients
for a (single) NFS server that are trused or allowed.

Cf. "Variables and functions must have names that
explain what they do, even if it makes them longer" in
https://github.com/rear/rear/wiki/Coding-Style#code-must-be-easy-to-read

Reply:
Ok, I agree. `NFS_SERVER_TRUSTED` sounded like it is the server that is being trusted and changing to `NFS_SERVERS_TRUSTED` would not fix that.

Reply:
I have renamed the variable to `NFS4SERVER_TRUSTED_CLIENTS` and the default value is `()`.

In the Prepare stage an error is thrown if the array is empty. Now the user is forced to set a trusted client.
@schlomo @jsmeix @pcahyna what do you think about?